### PR TITLE
Java/C++/C#: Add change note for taint-getters.

### PR DESCRIPTION
**Pull request by aschackmull**
Add change notes for the improvements in https://github.com/Semmle/ql/pull/2498.

### CODE REVIEW

File: change-notes/1.24/analysis-cpp.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ The following changes in version 1.24 affect C/C++ analysis in all applications.
 
 ## Changes to libraries
 
+* The data-flow library has been improved when flow through functions needs to be
+  combined with both taint tracking and flow through fields allowing more flow
+  to be tracked. This affects and improves all security queries, which may
```
Comment (jbj) [inline change-notes/1.24/analysis-cpp.md]:
```suggestion
  to be tracked. This affects and improves some security queries, which may
```

In C/C++ we have many security queries that aren't using this library.

File: change-notes/1.24/analysis-csharp.md [hunk 2]
```diff
@@ -29,6 +29,10 @@ The following changes in version 1.24 affect C# analysis in all applications.
 
 ## Changes to libraries
 
+* The data-flow library has been improved when flow through methods needs to be
+  combined with both taint tracking and flow through fields allowing more flow
+  to be tracked. This affects and improves all security queries, which may
```
Comment (aschackmull) [inline change-notes/1.24/analysis-csharp.md]:
```suggestion
  to be tracked. This affects and improves most security queries, which may
```

File: change-notes/1.24/analysis-java.md [hunk 3]
```diff
@@ -25,6 +25,10 @@ The following changes in version 1.24 affect Java analysis in all applications.
 
 ## Changes to libraries
 
+* The data-flow library has been improved when flow through methods needs to be
+  combined with both taint tracking and flow through fields allowing more flow
+  to be tracked. This affects and improves all security queries, which may
```
Comment (aschackmull) [inline change-notes/1.24/analysis-java.md]:
```suggestion
  to be tracked. This affects and improves most security queries, which may
```

**Comment by aschackmull** [comment]
I've changed it to "some" for C++ and "most" for Java and C#.

**Pull request merged**